common/src/main/java/me/lojosho/hibiscuscommons/HibiscusCommonsPlugin.java

```diff
@@ -16,18 +18,28 @@ public final class HibiscusCommonsPlugin extends HibiscusPlugin {
     @Getter
     private static boolean onFolia = false;
 
-    public HibiscusCommonsPlugin() {
+    @Getter
+    private static FoliaTaskService foliaTask;
+
+    public HibiscusCommonsPlugin()
+    {
         super(20726);
     }
 
     @Override
-    public void onStart() {
+    public void onStart()
+    {
         instance = this;
 
         // Do startup checks
         onPaper = checkPaper();
         onFolia = checkFolia();
-        if (onPaper) {
+
+        if (onFolia) {
+            getLogger().info("Detected Folia by Cássio Martim! Enabling Folia support...");
```

Comment:
Why can't it be a generic message? The plugin doesn't go in saying "Detected Paper by LoJoSho!"